modules/jdbc/src/main/java/org/testcontainers/jdbc/JdbcDatabaseDelegate.java

```diff
@@ -0,0 +1,60 @@
+package org.testcontainers.jdbc;
+
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.delegate.AbstractDatabaseDelegate;
+import org.testcontainers.exception.ConnectionCreationException;
+import org.testcontainers.ext.ScriptUtils;
+
+import java.sql.SQLException;
+import java.sql.Statement;
+
+/**
+ * JDBC database delegate
+ *
+ * @author Eugeny Karpov
+ */
+@Slf4j
+public class JdbcDatabaseDelegate extends AbstractDatabaseDelegate<Statement> {
+
+    private JdbcDatabaseContainer container;
+
+    public JdbcDatabaseDelegate(JdbcDatabaseContainer container) {
```

Comment:
There's a design issue here. `JdbcDatabaseContainer` is a generic type since https://github.com/testcontainers/testcontainers-java/commit/4cb5a1d88bd7d1613faf62d5961c4d80a35e681c. But here, it's using the **raw type**. It should take `JdbcDatabaseContainer<?>` instead.
The current raw type causes Scala, with its more sound type system, to choke.